lab-kyle/lib/hanoi.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
```

Comment:
You're close... Check out what I see when I run it:
```shell
> let towers = require('./towers')
undefined
> towers(3)
[ 9,
  [ { top: null, maxSize: null, size: 0 },
    { top: null, maxSize: null, size: 0 },
    { top: [Object], maxSize: null, size: 4 } ] ]
>
```

Looks like the size of the final stack is off by one?